setup.sh

```diff
@@ -38,8 +38,12 @@ if [[ "$OS" == "Fedora" ]]; then
 elif [[ "$OS" == "Ubuntu" ]] || [[ "$OS" == "LinuxMint" ]]; then
   sudo apt-get install ffmpeg python-imdbpy python-notify2
   sudo apt-get install python-dbus libssl-dev libffi-dev
-  sudo -H pip install -r requirements.txt
   sudo apt-get install chromium-chromedriver
+  sudo pip install virtualenv
+  virtualenv env
+  virtualenv -p /usr/bin/python2.7 env
+  source env/bin/activate
```

Comment:
it didn't work for me

Reply:
can you give me errors(screenshot or log)

Reply:
![error_message](https://user-images.githubusercontent.com/15054312/31576744-dd33ef88-b109-11e7-862e-5f9e719bd621.png)

Reply:
Is it working now?